<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QWidget" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit amount:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_amount">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest rate (%):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="interest_rate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Term:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="years">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>years</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="months">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>months</string>
   </property>
  </widget>
  <widget class="QComboBox" name="periodicity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>every day</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every week</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every quarter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every half year</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every year</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>at the end of the term</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="capitalization">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Capitalization</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Replenishment:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="replenishment_date">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="replenishment_periodicity">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>once</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every two months</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every quarter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every half year</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>once a year</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="replenishment_amount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Partial withdrawal:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="withdrawal_periodicity">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>once</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every two months</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every quarter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every half year</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>once a year</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="withdrawal_date">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="withdrawal_amount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Accrued interest:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit amount by the end of the term:</string>
   </property>
  </widget>
  <widget class="QLabel" name="accrued_interest">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="tax_amount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_deposit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>470</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="days">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>days</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_rate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax rate (%):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Beginning of the term:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="beginning">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="term">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
